src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -141,6 +142,25 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_RazorAddBuiltProjectOutputGroupOutput" Condition="'$(RazorCompileOnBuild)' == 'true'"  BeforeTargets="BuiltProjectOutputGroup">
+    <ItemGroup>
+      <BuiltProjectOutputGroupOutput Include="@(RazorIntermediateAssembly)" FinalOutputPath="$(Outdir)$(RazorTargetName).dll" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_RazorGetCopyToOutputDirectoryItems" AfterTargets="GetCopyToOutputDirectoryItems">
```

Comment:
I need to nuke this `AfterTargets`. We need to actually run at the end of `GetCopyToOutputDirectoryItemsDependsOn`

Reply:
FYI this doesn't interfere with the settings the control whether to copy output when **this** project is building. It only comes into play when you consider P2P references.

Reply:
Does this replace `_RazorCopyFilesToOutputDirectory`? Alternatively, should we also wire up `_RazorCopyFilesToOutputDirectory` to use the DependsOn?

Reply:
It does not replace `_RazorCopyFilesToOutputDirectory` - will change it to use the DependsOn